composer.json

```diff
@@ -44,6 +45,7 @@
     "lint": "phpcs",
     "lint-php8": "phpcs -p --standard=PHPCompatibilityWP --runtime-set testVersion 8.0- --extensions=php --ignore='vendor/,wordpress/,node_modules/' .",
     "test": "phpunit",
+    "test:watch": "phpunit-watcher watch < /dev/tty",
```

Comment:
It seems that `< /dev/tty` might not be necessary. I was able to start the watcher without it and it worked fine.